### PR TITLE
Task/create search component

### PR DESCRIPTION
**Pull request by a-w-m**
#63 #59

### CODE REVIEW

File: backend/src/app.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
 const express = require("express");
+
 //create express app
 const app = express();
 //require knex function to communicate with database
 const knex = require("./database/connection");
 
 //configure app
+//handle cors error (temporary fix) 
+ app.use(function(req, res, next) {
```
Comment (ChrisCodeCole) [inline backend/src/app.js]: app is now using cors from recent changes

File: backend/src/app.js [hunk 2]
```diff
@@ -21,6 +30,16 @@ app.get("/users", async (req, res) => {
 	}
 });
 
+app.get("/search/*", async (req, res) => {
```
Comment (ChrisCodeCole) [inline backend/src/app.js]: we can also move this one to a food truck route now that mimics the user route

File: backend/src/app.js (hunk 1 above)
Comment (ChrisCodeCole) [reply backend/src/app.js]: can most likely pull in changes from development and remove this section

Comment (a-w-m) [reply backend/src/app.js]: got it, I went ahead and removed my changes to app.js to resolve the conflicts.

**Review by ChrisCodeCole — APPROVED** [review]
Looks good!

**Pull request merged**